docs/flow/modin/experimental/core/execution/ray/implementations/pandas_on_ray/index.rst

```diff
@@ -0,0 +1,20 @@
+:orphan:
+
+ExperimentalPandasOnRay Execution
+=================================
+
+`ExperimentalPandasOnRay` execution keeps the underlying mechanisms of :doc:`PandasOnRay </flow/modin/core/execution/ray/implementations/pandas_on_ray/index>`
+execution architecturally unchanged and adds experimental features of ``Data Transformation``, ``Data Ingress`` and ``Data Egress`` (e.g. `read_pickle_distributed`).
+
+PandasOnRay and ExperimentalPandasOnRay differences
+---------------------------------------------------
+- another Factory ``PandasOnRayFactory`` -> ``ExperimentalPandasOnRayFactory``
```

Comment:
```suggestion
---------------------------------------------------

- another Factory ``PandasOnRayFactory`` -> ``ExperimentalPandasOnRayFactory``
```

Reply:
why the change is needed?

Reply:
I prefer to detach a section title and a respective description. That looks clearer.

Reply:
ok